config/default/TimerStrategyService.conf.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+use oat\taoQtiTest\models\runner\time\TimerStrategyService;
+
+return new TimerStrategyService();
```

Comment:
if timers defined in the taoTest and interface is used there, it would be better to move this there too